delphi/translators/for2py/floatNumpy.py

```diff
@@ -0,0 +1,149 @@
+from numpy import float32
```

Comment:
This file desperately needs comments indicating its purpose.  It's entirely possible that other people may read this code.

Reply:
Could you also please set a maximum line length of 80?